### PR TITLE
feat: investor-yearn

### PR DESCRIPTION
**Pull request by theoboldfrazier**

### CODE REVIEW

File: packages/investor-yearn/src/api/api.ts [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+  public vaults: Vault[]
+  private yearnSdk: Yearn<1>
+  private ssRouterContract: any
+
+  constructor({ adapter, providerUrl }: ConstructorArgs) {
+    this.adapter = adapter
+    this.provider = new Web3.providers.HttpProvider(providerUrl)
+    this.jsonRpcProvider = new JsonRpcProvider(providerUrl)
+    this.web3 = new Web3(this.provider)
+    this.yearnSdk = new Yearn(1, { provider: this.jsonRpcProvider, disableAllowlist: true })
```
Comment (0xdef1cafe) [inline packages/investor-yearn/src/api/api.ts:50]: please make network injectable here as we're spiking on testnets at the moment too

File: packages/investor-yearn/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# @shapeshiftoss/logger
+
+Shapeshift's yearn investor package.
```
Comment (0xdef1cafe) [inline packages/investor-yearn/README.md:3]:
```suggestion
ShapeShift's Yearn investor package.
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

**Comment by 0xdef1cafe** [comment]
looking good so far

File: packages/investor-yearn/src/api/api.ts (hunk 1 above)
Comment (theoboldfrazier) [reply packages/investor-yearn/src/api/api.ts]: Done.

**Review by 0xdef1cafe — APPROVED** [review]

**Review by DaoDev44 — APPROVED** [review]

File: packages/investor-yearn/README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# @shapeshiftoss/logger
```
Comment (gomesalexandre) [inline packages/investor-yearn/README.md:1]: 👀

**Review by gomesalexandre — CHANGES_REQUESTED** [review]

**Pull request merged**